src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -990,9 +990,11 @@ public void SetCookies(Uri uri, string cookieHeader)
         }
     }
 
+    // PathList needs to be public in order to maintain binary serialization compatibility as the System shim
+    // needs to have access to type-forward it.
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    internal struct PathList
+    public struct PathList
```

Comment:
Now that we're making this public in the implementation, does this mean we won't ever be able to change it back to being a class (to match desktop since it's now being boxed in the `Hashtable` anyway), or other changes to the public surface area that I made in #21074 (e.g. changing the enumerator return type, getting rid of the `Values` property, etc.)? Or should we make those changes now. Or make all the members internal/private now to allow for such changes in the future?